api-references/payments/prepaid-recharge.json

```diff
@@ -0,0 +1,1210 @@
+                          }
+                        }
+                      },
+                      "success": {
+                        "type": "boolean",
+                        "example": false
+                      },
+                      "trace_id": {
+                        "type": "string",
+                        "example": "98ade67b-89d8-4b55-80c8-6e41d25d3442"
+                      }
+                    }
+                  },
+                  "examples": {
+                    "500": {
+                      "value": {
+                        "error": {
+                          "code": "ERR999",
+                          "failureReason": "upstream-connectivity-error",
+                          "message": "Upstream API Connectivity Error"
+                        },
+                        "success": false,
+                        "trace_id": "98ade67b-89d8-4b55-80c8-6e41d25d3442"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+            ,
+            "parameters": [
+              {
+                "in": "header",
+                "name": "X-PRODUCT-INSTANCE-ID",
+                "schema": { "type": "string" },
+                "description": "This is provided by Setu and it represents a product instance for business in their system."
+              },
+              
+              {
+                "in": "body",
+                "name": "body",
+                "schema": {
+                  "type": "object",
+                  "required": [
+                    "customer",
+                    "mobile"
+                  ],
+                  "properties": {
+                    "mobile": {
+                      "type": "string",
+                      "example": "9192990013",
+                      "description": "The login mobile number of the initiating recharge"
+                    },
+                    "customer": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "name": {
+                            "type": "string",
+                            "example": "Mobile Number",
+                            "description": "The display name of the parameter."
+                          },
+                          "value": {
+                            "type": "string",
+                            "example": "7066573353",
+                            "description": "The value of the customer mobile number whose details are being fetched. "
+                          }
+                        }
+                      },
+                      "description": "The params for details for the mobile number."
+                    }
+                  }
+                }
+              }
+            ]
+          }
+        },
+        "/mobile-prepaid-recharge/fetch_plans":{
```

Comment:
Similar to above path comment, can this be `fetch-plans`? 

Let's drive consistency with our endpoints